src/components/ProfilePage/QuestCard.tsx

```diff
@@ -0,0 +1,215 @@
+import {
+  Box,
+  Container,
+  Stack,
+  Typography,
+  alpha,
+  useTheme,
+} from '@mui/material';
+import { useUserTracking } from 'src/hooks';
+import { Button } from '../Button';
+import DoneIcon from '@mui/icons-material/Done';
+import DateRangeRoundedIcon from '@mui/icons-material/DateRangeRounded';
+import {
+  QuestCardBottomBox,
+  QuestCardInfoBox,
+  QuestCardMainBox,
+  QuestCardTitleBox,
+} from './QuestCard.style';
+import { ProfilePageTypography } from './ProfilePage.style';
+import { XPBox } from './xpBox';
+
+interface QuestCardProps {
+  active?: boolean;
+  title?: string;
+  image?: string;
+  points?: number;
+  link?: string;
+  startDate?: string | null;
+  endDate?: string | null;
+  platformName?: string;
+  platformImage?: string;
+}
+
+function getDateFormat(startDate: string, endDate: string): string {
+  const sDate = new Date(startDate);
+  const eDate = new Date(endDate);
+  const startMonth = sDate.toLocaleString('default', { month: 'short' });
+  const endMonth = eDate.toLocaleString('default', { month: 'short' });
+  return `${startMonth} ${sDate.getDate()} - ${startMonth === endMonth ? '' : endMonth} ${eDate.getDate()}`;
+}
+
+export const QuestCard = ({
+  active,
+  title,
+  image,
+  points,
+  link,
+  startDate,
+  endDate,
+  platformName,
+  platformImage,
+}: QuestCardProps) => {
+  const theme = useTheme();
+  const { trackPageload, trackEvent } = useUserTracking();
+
+  return (
+    <QuestCardMainBox>
+      <img
+        src={image}
```

Comment:
Let´s use the formats Strapi provides for image-optimization. Maybe small would be a good fit. Right now it´s fetching an image bigger than 3000x3000px which is being rendered to only 240px:

![image](https://github.com/lifinance/jumper.exchange/assets/43956540/306540fd-0638-44dd-94f9-878fa1d4a00a)

Reply:
If small turns out to have a bad quality and does not look sharp, let´s find another size that works properly.